src/nanovg.c

```diff
@@ -1352,6 +1394,17 @@ static void nvg__flattenPaths(NVGcontext* ctx)
 			nvg__addPoint(ctx, p[0], p[1], NVG_PT_CORNER);
 			i += 3;
 			break;
+			case NVG_CURVETO:
+				curveCommands++;
+				p = &ctx->commands[i+1];
+				if (curveCommands > 3) {
+					cp1 = &ctx->commands[i - (3*3) + 1];
+					cp2 = &ctx->commands[i - (2*3) + 1];
+					cp3 = &ctx->commands[i - (1*3) + 1];
+					nvg__tesselateCurve(ctx, cp1[0], cp1[1], cp2[0], cp2[1], cp3[0], cp3[1], p[0], p[1], NVG_PT_CORNER);
```

Comment:
if i understand correctly this requires four curveto() commands to do anything. imho it would be good if you allowed the special case where curveCommands>2 and you use nvg__lastPoint() for the first point instead. this would allow move()curve()curve()curve(). 


in general, have you considered adding in the curve parameter to allow also distance parametrized and chordal variants of catmul rom? 

the cp1...3 names are misleading here imho. you're doing this to save variables?